geth/jail/cell_test.go

```diff
@@ -151,7 +151,10 @@ func (s *CellTestSuite) TestCellFetchErrorRace() {
 	// Find a free port in localhost
 	freeportListener, err := net.Listen("tcp", "127.0.0.1:0")
 	s.Require().NoError(err)
-	defer freeportListener.Close()
+	defer func() {
+		err := freeportListener.Close()
+		s.NoError(err)
```

Comment:
not a big deal, but why not to do check error on the same line

Reply:
I can, it looked like the error check was on a separate line in the majority of the codebase - if you feel strongly I can change this for all the error checks in this PR